### ISSUE TITLE
Enhance experience + features

### ISSUE THREAD

**Issue by ChialiTsai** [unlock-protocol/docs#33]
- [x] enable the search feature for reader
- [x] #38

Nice to have

- [x] adding "is this useful" at bottom of the article
- [x] change the dark/light mode to single toggle

**Comment by ChialiTsai** [comment]
Might want to look how we can use the notification bar to highlight some articles

**Comment by julien51** [comment]
> enable the search feature for reader

We can look at algolia? I can also create a bounty for that.. and maybe a community member can hook things up?

>  adding versioning dropdown to top nav (this is to help user navigate the release, versioning)
Maybe @searchableguy or @kalidiagne can help a bit?

**Comment by searchableguy** [comment]
Don't need much to do for enabling search. I will do it once everything else is done when merging the PR because I also need to test/check netlify CMS, replace and change some components, etc.

**Comment by ChialiTsai** [comment]
I just open a PR, https://github.com/unlock-protocol/docs/pull/32
this should cover 90% of update, will need some CSS assistance please see the other issue.

tagging @clemsos because he needs to test automation.

search can use plugin: https://docusaurus.io/docs/search

**Comment by ChialiTsai** [comment]
convert "versioning" to its own issue.

**Comment by wonderwomancode** [comment]
Added search as it's own issue #125 and "was this useful" button added #126

Closing this issue now.
